tests/integration/payment-gateway/SV_WC_Payment_Gateway_Payment_Token_Test.php

```diff
@@ -392,6 +392,74 @@ public function test_set_billing_hash() {
 	}
 
 
+	/**
+	 * @see Framework\SV_WC_Payment_Gateway_Payment_Token::get_environment()
+	 *
+	 * @dataProvider provider_get_enviroment
+	 */
+	public function test_get_environment( $stored_environment, $expected_enviroment ) {
+
+		$woocommerce_token = $this->get_new_woocommerce_credit_card_token();
+
+		$woocommerce_token->add_meta_data( 'environment', $stored_environment );
+
+		$token = new Framework\SV_WC_Payment_Gateway_Payment_Token( '12345', $woocommerce_token );
+
+		$this->assertSame( $expected_enviroment, $token->get_environment() );
+	}
+
+
+	/**
+	 * Provides test data for test_get_enviroment().
+	 *
+	 * @return array
+	 */
+	public function provider_get_enviroment() {
```

Comment:
```suggestion
	public function provider_get_environment() {
```

Reply:
🤦🏻‍♂️ - Thank you for catching those typos.